CMakeLists.txt

```diff
@@ -88,6 +88,7 @@ configure_file(config.h.in config.h)
 if(WIN32)
     install(FILES COPYING DESTINATION .)
     install(FILES README.md DESTINATION .)
+    install(FILES alsoft DESTINATION . RENAME alsoft.ini)
```

Comment:
Not sure where to place it on Linux.
```
# An additional configuration file (alsoft.ini on Windows, alsoft.conf on other
# OSs) can be placed alongside the process executable for app-specific config
# settings.
```
https://github.com/kcat/openal-soft/blob/master/alsoftrc.sample

Reply:
It's the best we can do, or not? Those few who really care can still modify this file, and the others won't take notice, because the sfx just sound as they always did.

Reply:
Ok, so we have to set ALSOFT_CONF in code, right? Like this:
```c
static char str[] = "ALSOFT_CONF=share/"PROJECT_SHORTNAME;
putenv(str);
```

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/981/commits/eadfa8944ca1312091219619254237fffd1a0a01